lib/AppInfo/Application.php

```diff
@@ -22,6 +30,21 @@ public function __construct() {
 	public function register(IRegistrationContext $context): void {
 		$context->registerEventListener(BeforeUserDeletedEvent::class, UserDeletedListener::class);
 		$context->registerEventListener(DatasourceEvent::class, AnalyticsDatasourceListener::class);
+		$context->registerEventListener(RenderReferenceEvent::class, TablesReferenceListener::class);
+
+		$context->registerSearchProvider(SearchTablesProvider::class);
+
+		$container = $this->getContainer();
+		/** @var IConfig $config */
+		try {
+			$config = $container->get(IConfig::class);
```

Comment:
```suggestion
		try {
			/** @var IConfig $config */
			$config = $container->get(IConfig::class);
```

The comment help the IDE to know which class the $config var is.